docs/design/data/armor/back.yaml

```
@@ -0,0 +1,44 @@
+source: docs/_incoming/XM Design Bible - Armor.tsv
+slot: Back
+tsv: |
+  Type	Level	Item Name	Source	DEF	HP	EP	ACC	ATK	EVA	MGT	PER	VIT	WIL	AGI	SOC	BST	BRW	ELM	HNT	KPR	MDR	NGT	TEC
+  Back	4	Harestride Mantle	Fabrication	1										x	x			x	x		x	x
+  Back	4	Harestride Mantle +1	Fabrication	2										x	x			x	x		x	x
+  Back	7	Etherwoven Cape	Fabrication	1												x			x	
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)